README.md

```diff
@@ -22,28 +22,51 @@ Basic usage for opening a SpringRoll application via a local path. This can be u
 </script>
 ```
 
-### Plugins
+## Plugins
 
-The Container supports several built-in plugins that mirror the state features in the application/game. These are initialized with HTML elements like buttons or input sliders.
+The Container supports several built-in plugins that mirror the state features in the application/game. These are initialized with HTMLElements (or selector strings) like buttons or input sliders.
 
-Here are some examples of the syntax for declaring plugins when creating the container. Not all (or any) plugins need to be included, only the features supported by the game.
+Here are some examples of the syntax for declaring plugins when creating the container. Not all (or any) plugins need to be included, only the features supported by the game. All examples use selector strings but plugins will also accept an HTMLElement as a parameter as well.
 
-PausePlugin, CaptionsPlugin, HelpPlugin:
+All plugins also accept multiple controls in the form of a selector string (see [Multiple Controls](#multiple-plugin-controls))
+
+### PausePlugin, HelpPlugin:
 ```javascript
-import { PausePlugin, CaptionsPlugin, HelpPlugin, Container } from 'springroll-container';
+import { PausePlugin, HelpPlugin, Container } from 'springroll-container';
 
   const container = new springroll.Container({
     iframeSelector: "#game",
     plugins: [
       //all three plugins here expect an HTML Button Element and take a single selector string
       new PausePlugin('#pause-button-selector'), //Pauses or unpauses the game
-      new CaptionsPlugin('#caption-button-selector'), //enables or disables captions
       new HelpPlugin('#help-button-selector'), //requests a hint or help from the game
     ]
+  });
+  container.openPath('game.html');
+```
+PausePlugin sets a className of 'paused' or 'unpaused' on individual pause buttons.
+
+### CaptionsPlugin:
+```javascript
+import { CaptionsPlugin, Container } from 'springroll-container';
+
+  const container = new springroll.Container({
+    iframeSelector: "#game",
+    plugins: [
+      new CaptionsPlugin({
+        captionsButtons: '#caption-button-selector',
+        //the three following options control caption styles, as radio buttons using the group name is ideal for selecting multiple radio buttons.
+        fontSizeRadios: 'input[name=font-size-radio-name]',  //expects exactly three(3) radio buttons for sm, md, and lg font sizes
```

Comment:
We should clarify that the _values_ should be `sm`, `md`, and `lg`:

```javascript
...
// expects exactly three radio buttons with values "sm", "md", and "lg"
fontSizeRadios: 'input[name=font-size-radio-name]',
...
```

It might be best to put that comment on a different line as well just because it's so long. I'd like to avoid horizontal scrolling if possible.